src/constants/CaseConverterEnum.ts

```diff
@@ -18,12 +18,13 @@ enum CaseConverterEnum {
    * ```
    */
   None = '(noCase)',
+  NoneUnderscore = 'noCase__',
```

Comment:
I am kinda thinking PascalCase would be nicer for this new ones?

```suggestion
  NoneUnderscore = 'NoCase__',
```

```js
__replacerSlot__NoCase__
```

Reply:
Sounds good, think I got them all